Model/Config.php

```diff
@@ -139,14 +153,22 @@ public function getServiceId()
         return trim($this->store->getConfig('payment/paynl/serviceid'));
     }
 
-    public function getIconUrl() {
-        $url = 'https://static.pay.nl/payment_profiles/50x32/#paymentOptionId#.png';
-        $iconUrl = trim($this->store->getConfig('payment/paynl/iconurl'));
+    public function getIconUrl($PaymentMethodeID)
+    {      
+        $iconUrl = 'https://static.pay.nl/payment_profiles/50x32/#paymentOptionId#.png';
+        $configured = $this->configureSDK();
+        if ($configured) {
+            $list = Paymentmethods::getList();
+            if (isset($list[$PaymentMethodeID])) {
+                $iconUrl = $this->resources->getViewFileUrl("Paynl_Payment::logos/" . $list[$PaymentMethodeID]['brand']['id'] . ".png");
```

Comment:
Kun je hier ook checken of de keys 'brand' en 'id' ook geset zijn? En variabelen niet met hoofdletter beginnen.